Maquetado/index.html

```diff
@@ -14,6 +14,14 @@
     <p class='text'>Front-End Tools</p>
   </div>
 
+  <div class="tech">
+    <div class="content-tech">
+      <img class="img">
+      <h2 class="h2">ReacJS</h2>
+      <h3 class="h3">Release:2013</h3>
+      <p class="text">Reac is a declarative, component-based JavaScript library for building user interfaces</p>
```

Comment:
You should use more descriptive class names. Like tech, tech-title, tech-release-date, tech-text- tech-image